### DISCUSSION TITLE
Tuner jumps from frequency to frequency (P25P1)

### DISCUSSION THREAD

**Discussion by ah1102** [DSheirer/sdrtrunk#1533] [Ideas]
In P25P1 mode, the tuner's center frequency jumps periodically while trying to track channels. But my tuner is enough to capture all the channels.

Is it possible to do something, for example, force the center frequency of the tuner to be fixed?

Or automatically programmatically track the need for such jumps in the center frequency of the tuner.

![Снимок экрана 2023-04-24 в 20 23 58](https://user-images.githubusercontent.com/58341369/234075409-9f5c260f-2573-400d-8dda-1439c4d4672a.png)

**Comment by ErikSwan** [comment]
I recently asked basically the same question here: https://groups.google.com/g/sdrtrunk/c/ZRUmLbAgMX8

One workaround (as described in that thread) is to manually place two dummy NBFM channels on either side of the range where you want to keep the tuner fixed. That should stop the automatic re-tuning that takes place. I never got a clear answer as to why sdrtrunk re-tunes even when it seems like it isn't necessary (e.g. when all voice frequencies already fit within the current sampling/tuning window of the SDR).

Also, as an FYI and for future reference, it seems like at the moment the Google Groups forum is more active than the "Discussions" here on GitHub (a little unfortunate as I prefer GitHub and having two places for discussion scatters the conversation, but that's the way it is right now).
